renderer.py

```diff
@@ -0,0 +1,226 @@
+import io
+import utils
+
+from colorama import Fore, Style
+from datetime import datetime, timedelta
+
+# from cliStocksTracker import Portfolio
+
+
+class Renderer(metaclass=utils.Singleton):
```

Comment:
this class is for all printing - it conforms all rounding to the utils implementation